deploy/olm-catalog/unifiedpush-operator/0.2.0/unifiedpush-operator.v0.2.0.clusterserviceversion.yaml

```diff
@@ -190,9 +200,6 @@ spec:
                 - unifiedpush-operator
                 env:
                 - name: WATCH_NAMESPACE
-                  valueFrom:
-                    fieldRef:
-                      fieldPath: metadata.annotations['olm.targetNamespaces']
                 - name: APP_NAMESPACES
                   value: unifiedpush-apps
```

Comment:
Is it possible to override this value using OLM? We'd need to provide the MDC namespace in the case of integreatly?

Reply:
I'm not too sure on this one. @matskiv may be able to better comment on this

Reply:
Good catch. I don't know how to override that using OLM.
We should be able to just patch it from integreatly-operator, but that might cause other issues (e.g. OLM might revert it on upgrade?).

Reply:
These annotations might help: https://docs.openshift.com/dedicated/4/applications/operators/olm-understanding-olm.html#olm-operatorgroups-csv-annotations_olm-understanding-olm

Alternatively, I'm happy to have this set to `unifiedpush-apps,mobile-developer-console,openshift-mobile-developer-console` by default.

How does that sound @laurafitzgerald?